backend/app/tests/anti_recommenders/conftest.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="module")
```

Comment:
Nit: I'd probably just put this skip on the openai_normal_anti_recommender fixture.

Reply:
That makes more sense. I will change this.